test/fuzzTesting/fuzzHelper.cpp

```diff
@@ -19,12 +19,12 @@
  * @date 2015
  */
 
-#include "fuzzHelper.h"
-
 #include <chrono>
 #include <boost/random.hpp>
 #include <boost/filesystem/path.hpp>
 #include <libevmcore/Instruction.h>
+#include <test/libtesteth/TestOutputHelper.h>
+#include <test/fuzzTesting/fuzzHelper.h>
```

Comment:
There is a rule to include matching header (name.cpp -> name.hpp) as the first include (as it was here). I think we should keep it.